automation/build.sh

```diff
@@ -21,9 +21,6 @@ rpmbuild \
     -D "_topmdir $ROOT_PATH/rpmbuild" \
     -ts ./*.gz
 
-# install any build requirements
-yum-builddep $ROOT_PATH/output/*src.rpm
-
```

Comment:
I'd leave it here, without it you cannot add new dependencies to the project and have successfull build

Reply:
No, buildcontainer can provide new dependencies only after the PR adding them is merged and buildcontainer rebuilt. Using above will allow you to test the new dependency change even before it's merged

Reply:
that chicken and egg problem was the reason for https://github.com/oVirt/buildcontainer/pull/14 :)
no need to merge any temporary code anywhere. gets overwritten nightly.